test/parallel/test-benchmark-path.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+require('../common');
+
+// Minimal test for path benchmarks. This makes sure the benchmarks aren't
+// horribly broken but nothing more than that.
+
+const assert = require('assert');
+const fork = require('child_process').fork;
+const path = require('path');
+
+const runjs = path.join(__dirname, '..', '..', 'benchmark', 'run.js');
+const argv = ['--set', 'n=1',
+              '--set', 'path=',
+              '--set', 'pathext=',
+              '--set', 'paths=',
+              'path'];
```

Comment:
The two benchmarks that have `props` each seem to have a single entry only. Are you suggesting adding `props=` to future-proof things in case more entries are added later to the benchmarks? (Seems like a good idea to me. Making sure I'm not misunderstanding your suggestion.)